### ISSUE TITLE
Go binary deployed as +dirty

### ISSUE THREAD

**Issue by dwbrown2** [kubecost/cost-analyzer-helm-chart#16]
The go binary in the production docker images is showing up as `+dirty`?

```# docker run --rm -it ajaytripathy/kubecost-cost-model:prod-1.18.2
I0430 19:44:55.279596       1 main.go:202] Starting cost-model (git commit "bd779830c98be5b101f2d9f0fe9b1e1f1fcea78f+dirty")```

**Comment by dwbrown2** [comment]
Pretty sure this is being done to reduce build times, but I'll let @AjayTripathy comment in more detail.

**Comment by mdaniel** [comment]
`I0521 00:40:41.405012       1 main.go:279] Starting cost-model (git commit "c1813c6425d90758e6dafe75dca1bc97b2313502")`
this appears to be fixed as of 1.24.0 :tada:

**Comment by AjayTripathy** [comment]
We're now building / deploying from master directly so this shouldn't be an issue.
